frb_codegen/src/generator/dart/ty.rs

```diff
@@ -31,6 +31,7 @@ macro_rules! type_dart_generator_struct {
             pub ir: $ir_cls,
             pub context: TypeGeneratorContext<'a>,
             pub dart_api_class_name: Option<String>,
+            pub is_exception: bool,
```

Comment:
p.s. "can we remove the is_exception field from dart generator struct" - means this field